clang/lib/Analysis/FlowSensitive/SmartPointerAccessorCaching.cpp

```diff
@@ -71,7 +71,7 @@ bool hasSmartPointerClassShape(const CXXRecordDecl &RD, bool &HasGet,
                                 ->getCanonicalTypeUnqualified();
         }
       }
-    }
+    } break;
```

Comment:
newline?

Reply:
clang-format wants to put it on that same line =/

Reply:
Could put it inside the `case : { ...}` scope, it might be ok with that.

Reply:
On second thought, there are a ton of these in the codebase already. It's fine as is.